NEWS

```diff
@@ -2,6 +2,21 @@
 
 Sandia OpenSHMEM NEWS -- history of user-visible changes.
 
+v1.3.4
+------
+- Added manpages for OpenSHMEM API routines.
+- Added polling limit control variables SMA_OFI_TX_POLL_LIMIT and
+  SMA_OFI_RX_POLL_LIMIT (see README for details) to allow control over quiet
+  and wait polling limits, respectively.
+- Rewrite of environment variable management code and improved info output.
+- Added support for processor atomics in thread-safe build.
+- Added support for local communication optimization (--enable-memcpy).
+- Added support for libfabric FI_MR_RMA_EVENT memory registration mode.
+- Fixed completion race in multithreaded mode.
+- Fixed bugs in support for zero-length point-to-point and collective
+  communication.
+- Multiple bugfixes to test suite, including GUPS benchmark
+
```

Comment:
Looks great - a few extras that may or may not warrant inclusion:
PMIx/OpenMPI launcher? (but, it's experimental)
Mellanox tests? (but, not user-facing...)